src/Volume.ts

```diff
@@ -112,32 +140,34 @@ export default class Volume {
   private pixel_size: [number, number, number];
   /* eslint-enable @typescript-eslint/naming-convention */
 
-  constructor(imageInfo: ImageInfo) {
+  constructor(imageInfo?: ImageInfo) {
     this.scale = new Vector3(1, 1, 1);
     this.currentScale = new Vector3(1, 1, 1);
     this.physicalSize = new Vector3(1, 1, 1);
     this.normalizedPhysicalSize = new Vector3(1, 1, 1);
 
     this.loaded = false;
-    this.imageInfo = imageInfo;
-    this.name = imageInfo.name;
+    this.imageInfo = imageInfo || defaultImageInfo;
```

Comment:
is there a typescript way to say that defaultImageInfo is the defaulted constructor parameter?
`constructor(imageInfo: ImageInfo = defaultImageInfo)`
and then you can just do `this.imageInfo = imageInfo`
Not a big difference I guess...

Reply:
I'm still slightly concerned about the Object.freeze for the default parameter.  It would be much safer if we just instantiated a new one every time.